packages/turf-rhumb-destination/index.js

```diff
@@ -39,7 +45,7 @@ function rhumbDestination(origin, distance, bearing, options) {
     // compensate the crossing of the 180th meridian (https://macwright.org/2016/09/26/the-180th-meridian.html)
     // solution from https://github.com/mapbox/mapbox-gl-js/issues/3250#issuecomment-294887678
     destination[0] += (destination[0] - coords[0] > 180) ? -360 : (coords[0] - destination[0] > 180) ? 360 : 0;
-    return point(destination);
+    return point(destination, properties);
```

Comment:
Proper properties handling will be done in `@turf/helpers`.

Something like this will be added

```js
for (var prop in properties) {
    pt.properties[prop] = properties[prop];
}
```